ext/nrf/cc310_glue.c

```diff
@@ -1,7 +1,20 @@
 /*
- * Copyright (c) 2019 Nordic Semiconductor ASA
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
We do need an SPDX-License-Identifier for the Apache license. In fact, none of the rest of the text is needed (but is up to you).

Reply:
@sigvartmh Can you address the issue?

Reply:
Done sorry was a bit late .

Reply:
@d3zd3z Can you take a look again?